features/poller/remote/src/main/java/org/opennms/netmgt/poller/remote/Poller.java

```diff
@@ -147,13 +151,20 @@ private void schedulePolls() throws Exception {
 
             m_pollerFrontEnd.setInitialPollTime(polledService.getServiceId(), initialPollTime);
 
-            Trigger pollTrigger = new PolledServiceTrigger(polledService);
-            pollTrigger.setStartTime(initialPollTime);
+            /*
+            JobBuilder.newJob(PollJob.class)
+                .withIdentity(jobName)
+                .usingJobData(PollJobDetail.JOB_DATA_MAP_KEY_POLLEDSERVICE, polledService)
+                .usingJobData(PollJobDetail.JOB_DATA_MAP_KEY_POLLERFRONTEND, m_pollerFrontEnd);
+            */
```

Comment:
This is an example of using the new Quartz 2 JobBuilder API. I didn't have a way to test whether it was correct so I left it commented.